spec/models/registration_spec.rb

```diff
@@ -91,22 +91,14 @@
     end
   end
 
-  describe '#destroy_purchased_tickets' do
-    it 'destroys purchased tickets if tickets are purchased' do
+  describe 'does not destroy purchased tickets' do
```

Comment:
I think this is no longer a model test and should be moved to a feature test. @ChrisBr ?